### DISCUSSION TITLE
v3.0.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#7732] [Releases & Announcements]
We're thrilled to announce the first stable candidate for `v3.0`, paving the way for exciting new features. With this update, you'll gain immediate access to enhancements such as `Dark Mode`, setting the stage for more to come.

Thanks to @jordan-brough, @civilcoder55, @ljcp, @memorycancel, @clairton @karthi07, @LiamAshdown, @merlijndetovenaar84, @nhippenmeyer, @Jatin-MYST, @CristianDuta

[Join the V3 Discussion](https://github.com/orgs/chatwoot/discussions/7570)

### ⚠️ Important Note for Integration Testing:
If you have built integrations over the 2.x versions of Chatwoot, please ensure to test them thoroughly before updating to 3.x versions. We are committed to providing seamless transitions, and we'll be updating our developer documentation in the coming days to guide you through the process.

### 📝 What's Next?
Stay tuned for updates, patches, and a closer look at what's in store for future versions. We value your feedback and contributions, so please feel free to engage with our team and community.

Thank you for being a part of the Chatwoot journey, and happy coding!

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v3.0.0'>v3.0.0</a>.</em>

**Comment by jochenstu** [comment]
Fair warning, I just upgraded to v3.0 (using cwctl --upgrade) and, among other things, my inboxes are gone. Trying to sort this out, but you might wanna hold out.

**Reply by pranavrajs** [reply]
@jochenstu Can you please share the error logs if there is any?

**Reply by jochenstu** [reply]
@pranavrajs This got fixed, first by a) getting a full compile using node 16 (over 18) https://github.com/chatwoot/chatwoot/issues/7741 and then b) by clearing cookies https://github.com/chatwoot/chatwoot/issues/7744

Thanks for picking this up though, people have always been very reactive and helpful.

**Comment by sanjayws** [comment]
Thanks and congratulations to this release. 

Would the v2 branch be maintained here on?

**Reply by pranavrajs** [reply]
There is no long term support for a specific version. We expect everyone to upgrade to the latest version as soon as possible.

**Reply by cruz4d3r** [reply]
Excuse my ignorance, but I have tried to update from 2.18 to 3.0 but I have not been successful, do you have any guide to do it? Thank you.

**Reply by pranavrajs** [reply]
@cruz4d3r It should be straightforward. Do you see any errors?

**Reply by cruz4d3r** [reply]
It has no bugs, but the interface is not updated and has version 3.0.0. I have 4 ram and 4 processors in the vps. The update process was manually with the guide that you put in this thread.

**Reply by Jatin-MYST** [reply]
@cruz4d3r Yeah, I had that issue before.
The problem is that compiling the interface doesn't clean the previous version first.
Use this command that does the cleaning then recompiling:
`RAILS_ENV=production rake assets:clean assets:clobber assets:precompile && sudo systemctl restart chatwoot.target`
